mslib/mscolab/server.py

```diff
@@ -489,13 +491,17 @@ def get_operation_details():
 @verify_user
 def set_last_used():
     op_id = request.form.get('op_id', None)
+    days_ago = int(request.form.get('days', 0))
     operation = Operation.query.filter_by(id=int(op_id)).first()
-    operation.last_used = datetime.datetime.utcnow()
+    operation.last_used = datetime.datetime.utcnow() - datetime.timedelta(days=days_ago)
     temp_operation_active = operation.active
-    operation.active = True
+    if days_ago > ARCHIVE_THRESHOLD:
+        operation.active = False
+    else:
+        operation.active = True
     db.session.commit()
     # Reload Operation List
-    if not temp_operation_active:
+    if temp_operation_active != operation.active:
```

Comment:
the wording maybe can be changed to archived, unarchived etc.

Reply:
changing operation.active requires changes to the DB, I guess?

Reply:
I am not convinced that we should change the DB format to be more consistent in a part that is not user-visible.